osdep/terminal-unix.c

```diff
@@ -437,10 +422,19 @@ static MP_THREAD_VOID terminal_thread(void *ptr)
         if (fds[1].revents & POLLIN) {
             int8_t c = -1;
             (void)read(stop_cont_pipe[0], &c, 1);
-            if (c == PIPE_STOP)
+            if (c == PIPE_STOP) {
                 do_deactivate_getch2();
-            else if (c == PIPE_CONT)
+                if (isatty(STDERR_FILENO)) {
+                    (void)write(STDERR_FILENO, TERM_ESC_RESTORE_CURSOR,
+                                sizeof(TERM_ESC_RESTORE_CURSOR) - 1);
+                }
+                // make do with SIGSTOP since it's effectively the same and
+                // there's no way to get a function pointer to the "real
```

Comment:
nit: I think those handlers actually live in the kernel so while "no way to get a function ptr" is technically correct it also doesn't make sense.
I suggest rewording this to "no way to explicitly invoke the default signal handler"

Reply:
Done